dicom_util/src/main/java/com/google/cloud/healthcare/DicomWebValidation.java

```diff
@@ -0,0 +1,42 @@
+package com.google.cloud.healthcare;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.regex.Pattern;
+
+public class DicomWebValidation {
+    private static Logger log = LoggerFactory.getLogger(DicomWebValidation.class);
+
+    private static String DICOMWEB_PATH = "https:\\/\\/healthcare.googleapis.com\\/.*?\\/projects\\/.*?\\/locations\\/.*?\\/datasets\\/.*?\\/dicomStores\\/.*?\\/dicomWeb";
+    private static String STUDIES = "studies";
+    private static String STUDIES_PATH = DICOMWEB_PATH + "\\/" + STUDIES;
+    private static String HEALTHCARE_API_ROOT = "https://healthcare.googleapis.com";
+
+    public static ValidationPattern DICOMWEB_ROOT_VALIDATION =
+            new ValidationPattern(Pattern.compile(DICOMWEB_PATH), "Google Healthcare Api dicomWeb root path");
+    public static ValidationPattern DICOMWEB_STUDIES_VALIDATION =
+            new ValidationPattern(Pattern.compile(STUDIES_PATH), "Google Healthcare Api dicomWeb studies path");
+
+    public static void validatePath(String path, ValidationPattern validation){
```

Comment:
Can you please add a couple of unit tests for this class?

Reply:
Ok, added.